cms/templates/settings.html

```diff
@@ -226,7 +228,7 @@ <h2 class="title-2">${_('Course Schedule')}</h2>
               </div>
             </li>
             <%
-              enrollment_end_readonly = "readonly aria-readonly=\"true\"" if not enrollment_end_editable else ""
+              enrollment_end_readonly = HTML("readonly aria-readonly=\"true\"") if not enrollment_end_editable else ""
```

Comment:
@mushtaqak This was the cause of the Python unit test failure (cms.djangoapps.contentstore.tests.test_course_settings.CourseEnrollmentEndFieldTest.test_course_details_with_enabled_setting_non_global_staff).